### DISCUSSION TITLE
MSCS%40Purdue/

### DISCUSSION THREAD

**Discussion by giscus** [opencsapp/opencsapp.github.io#297] [Announcements]
# MSCS%40Purdue/

None

https://opencs.app/MSCS%40Purdue/

**Comment by Met4physics** [comment]
这个是为什么

**Comment by 10-OASIS-01** [comment]
Purdue为什么时陆本黑洞呀

**Reply by notij** [reply]
Purdue新校长是个华裔 不是很亲华， 他本身就是读ece的，他之前是国务卿的科学和技术顾问， 你品品
